### DISCUSSION TITLE
Upgrade Signoz to current latest v0.18.3

### DISCUSSION THREAD

**Discussion by resulraveendran** [SigNoz/signoz#2690] [Q&A]
The current deployed version is v0.7.2 across all the projects. Since we need to do multi version migration process along with persisting the data in the clickhouse database . What are the steps i need tofollow for this?

**Comment by resulraveendran** [comment]
To Upgrade till V.12.0 we got from this document (https://signoz.io/docs/operate/migration/upgrade-0.12/). Is there any document for upgrade to v0.18.3 or what are the steps i need to follow for Upgrade?

**Comment by srikanthccv** [comment]
The schema migrations are handled automatically, and there were no new table data migrations so you should be able to migrate from `0.12` to `0.18` without any issues.
